libkbfs/folder_branch_ops.go

```diff
@@ -1014,20 +1005,20 @@ func (fbo *folderBranchOps) nowUnixNano() int64 {
 }
 
 func (fbo *folderBranchOps) maybeUnembedAndPutOneBlock(ctx context.Context,
-	md *RootMetadata) error {
+	md *RootMetadata) (*blockPutState, error) {
```

Comment:
this function is now misnamed, right? Shouldn't it be `maybeUnembedAndPutBlocks`?